configurations/test/env.yml

```diff
@@ -1,4 +1,6 @@
 AUTH0_DOMAIN: test.auth0.com
 AUTH0_API_CLIENT: test-auth0-client-id
 AUTH0_API_SECRET: test-auth0-secret
-MONGO_DB_NAME: otp_middleware_test
\ No newline at end of file
+MONGO_DB_NAME: otp_middleware_test
+OTP_SERVER: https://fdot-otp-server.ibi-transit.com
```

Comment:
We should try to avoid storing actual live endpoints in test config. Let's keep this `http://www.example.com`.

Reply:
Parameters no longer required. Removed.